docs/dev/run_the_etl.rst

```diff
@@ -24,15 +23,111 @@ There are two main scripts that are involved in the PUDL processing pipeline:
+"full" or "fast" conditions. You can copy, rename, and modify these files to suit your
+needs. Updating the ``name``, ``title``, and ``description`` fields will
+ensure that new ``pudl-etl`` script outputs won't override old ones. The layout of
+these files is depicted below:
+
+.. code-block::
+
+      # FERC1 to SQLite settings
+      ferc1_to_sqlite_settings:
+        ├── dataset f2s parameter (e.g. tables) : editable list of tables
+        ├── dataset f2s parameter (e.g. years) : editable list of years
+
+      # PUDL ETL settings
+      name : unique name identifying the etl outputs
+      title : short human readable title for the etl outputs
+      description : a longer description of the etl outputs
+      datasets:
+        ├── dataset name
+        │    ├── dataset etl parameter (e.g. tables) : editable list of tables
+        │    └── dataset etl parameter (e.g. years) : editable list of years
+        └── dataset name
+        │    ├── dataset etl parameter (e.g. states) : editable list of states
+        │    └── dataset etl parameter (e.g. years) : editable list of years
+
+.. note::
+
+    Do not change anything other than the dataset parameters and the name, title, and
+    description fields unless you want to remove an entire dataset. For example, CEMS
+    data takes a long time to load so you can comment out or delete all settings
+    pertaining to CEMS. See below for a way to add it later.
+
+Both scripts enable you to choose which **years** and **tables** you want to include:
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Parameter
+     - Description
+   * - ``years``
+     - A list of years to be included in the FERC Form 1 Raw DB or the PUDL DB. You
+       should only use a continuous range of years. Check the :doc:`/data_sources/index`
+       pages for the earliest available years.
+   * - ``tables``
+     - A list of strings indicating what tables to load. The list of acceptable
+       tables can be found in the the example settings file. **We recommend including
+       all the tables for a given dataset due to dependences. The impact on runtime is
+       negligible.**
+
+The ``pudl_etl`` script CEMS data does not allow you to select which tables to include
+(it's just the one) but it does allow you to select **years** and **states**.
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Parameter
+     - Description
+   * - ``years``
+     - A list of the years you'd like to process CEMS data for. You should
+       only use a continuous range of years. Check the :doc:`/data_sources/epacems` page
+       for the earliest available years.
+   * - ``states``
+     - A list of the state codes you'd like to process CEMS data for. You can specify
+       ``all`` if you want to process data for all states. This may take a while!
 
 .. seealso::
 
-    * :ref:`install-workspace` for more on how to create a PUDL data workspace.
```

Comment:
Did you mean to remove this link to the `install-workspace` page?

Reply:
Yeah, I added the link up higher at the beginning of the Settings section in line 32